docs/source/howto/interact.rst

```diff
@@ -0,0 +1,112 @@
+.. _how-to:interact:
+
+**************************
+How to interact with AiiDA
+**************************
+
+There are a variety of manners to interact with AiiDA:
+
+ * :ref:`Through the command line interface <how-to:interact-cli>`
+ * :ref:`Through interactive shells <how-to:interact-shell>`
+ * :ref:`Through interactive notebooks <how-to:interact-notebook>`
+ * :ref:`Through scripts <how-to:interact-scripts>`
+ * :ref:`Through the REST API <how-to:interact-restapi>`
+
+
+.. _how-to:interact-cli:
+
+Command line interface
+======================
+
+AiiDA comes with a command line interface called ``verdi``.
+The :ref:`reference:command-line` section gives an overview of all available commands.
+For more detailed information, refer to the topic section :ref:`topics:cli`.
+
+
+.. _how-to:interact-shell:
+
+Interactive shells
+==================
+
+AiiDA provides a Python API that can be used from an interactive shell, such as `IPython <https://ipython.org/>`_.
+To use AiiDA from an IPython shell, one just has to load the profile as follows:
+
+.. code-block:: ipython
+
+    In [1]: from aiida import load_profile
+
+    In [2]: load_profile()
+    Out[2]: <aiida.manage.configuration.profile.Profile at 0x7fccfd6c50a0>
+
+One can pass a particular profile name to :meth:`~aiida.load_profile`, otherwise the default profile is loaded.
+Alternatively, the command line interface ``verdi`` provides a handy shortcut:
+
+.. code-block:: console
+
+    $ verdi shell
+
+This command will open a normal IPython shell but automatically loads the AiiDA profile.
+In addition, certain modules from AiiDA's API that are used very often are automatically imported.
```

Comment:
There used to be a documentation of this, is this still available, or should we put an incomplete list here at least to give a feeling to users of what is auto-imported? Otherwise, pointing to https://github.com/aiidateam/aiida-core/blob/2174924e40c3c9f00263f2e892e63a07eb90b40f/aiida/cmdline/utils/shell.py#L12 
could be an option.

Also, if no docs for `verdi shell` exists, we should probably add (here or a specific page) on this, at least to  mention the `verdi.shell.auto_import` option of `verdi config`